include/zephyr/bluetooth/audio/audio.h

```diff
@@ -222,6 +222,7 @@ struct bt_codec_data {
  * These values are defined by the Generic Audio Assigned Numbers, bluetooth.com
  */
 enum bt_audio_location {
+	BT_AUDIO_LOCATION_NOT_ALLOWED = 0,
```

Comment:
Nitpick, but I'd suggest `BT_AUDIO_LOCATION_PROHIBITED`, to follow the same naming as `BT_AUDIO_CONTEXT_TYPE_PROHIBITED`

Reply:
Done